packages/angular/build/src/utils/server-rendering/manifest.ts

```diff
@@ -149,9 +139,7 @@ export function generateAngularServerAppManifest(
       file.path === INDEX_HTML_CSR ||
       (inlineCriticalCss && file.path.endsWith('.css'))
     ) {
-      serverAssetsContent.push(
-        `['${file.path}', async () => ${escapeUnsafeChars(JSON.stringify(file.text))}]`,
-      );
+      serverAssetsContent.push(`['${file.path}', async () => \`${escapeUnsafeChars(file.text)}\`]`);
```

Comment:
I think that using string literals here makes the other replacements unnecessary. However, the code scanner does not run on pull requests.